src/modules/settings/components/Window.jsx

```diff
@@ -28,7 +27,7 @@ function Page(props) {
   }
 }
 
-const validateModal = () => window.innerWidth < 600 && isPopout();
```

Comment:
i think we should keep the popout, but also allow it if it's a chat page.. maybe check `window.location.pathname.endsWith('/chat')`